packaging/rpm/rear.spec

```diff
@@ -154,6 +156,9 @@ fi
 %prep
 %setup -q
 
+%check
```

Comment:
Cosmetic remark: please move the `%check` section after `%install`. rpmbuild executes `%check` after `%install`, so it is more readable if in the spec file they follow in the same order.

Reply:
I moved it.